src/main/java/org/zalando/nakadi/domain/EventTypeSchema.java

```diff
@@ -37,6 +46,24 @@ public void setSchema(final String schema) {
         this.schema = schema;
     }
 
+    @Nullable
```

Comment:
@v-stepanov Correct. I think it should be like so. If a null shows up it means that there is a bug in the code. It should not be treated. NullPointer whould raise and we should fix the root cause.